cmd/neofs-cli/modules/container/util.go

```diff
@@ -12,8 +12,6 @@ import (
 
 const (
 	attributeDelimiter = "="
-
-	awaitTimeout = 120 // in seconds
```

Comment:
This is still relevant as a concept. Commands should have longer default timeouts if invoked with `--await`.

Reply:
i don't see what's the relevance. Timeout controls time period of command execution regardless of the set of internal actions. As a user, I'd expect unconditional default timeout since it's easier to understand and I can't provide await timeout explicitly

Reply:
Timeouts can be different for different actions and in this case they should be. 15s is just not enough in many `--await` cases (consider mainnet). This doesn't introduce any new timeout, it's the same old one, just a different default value (XXX without `--await` and YYY with `--await`, pretty easy). The effect of `--await` is felt by users immediately, so you won't need to explain anything, commands need a bit more time in this case.

Reply:
i got it. Suggest to leave `15s` for "simple" ops and `1m` for async-await, thoughts?

Reply:
OK for me.

Reply:
done